thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobResource.java

```diff
@@ -289,15 +290,15 @@ public Response tuneAlertFilter(@PathParam("function_id") String id,
     // create alert filter and evaluator
     AlertFilter alertFilter = alertFilterFactory.fromSpec(anomalyFunctionSpec.getAlertFilter());
     AlertFilterUtil evaluator = new AlertFilterUtil(alertFilter);
+
+    // create alert filter auto tune
+    AlertFilterAutoTune alertFilterAutotune = alertFilterAutotuneFactory.fromSpec(autoTuneType);
+    LOG.info("initiated alertFilterAutoTune of Type {}", alertFilterAutotune.getClass().toString());
     try {
       //evaluate current alert filter (calculate current precision and recall)
-      double[] evals = evaluator.getEvalResults(anomalyResultDTOS);
+      Double[] evals = evaluator.getEvalResults(anomalyResultDTOS);
       LOG.info("AlertFilter of Type {}", alertFilter.getClass().toString(), "has been evaluated with precision: {}", evals[AlertFilterUtil.PRECISION_INDEX], "recall:", evals[AlertFilterUtil.RECALL_INDEX]);
```

Comment:
This logger statement does not look right.
Can you please confirm if it prints log info as expected ?